### PR TITLE
Add validation to order controller

### PR DESCRIPTION
**Pull request by hellodit**
Hi, I just add some validation to your order feature. I think it should be more secure.

### CODE REVIEW

**Comment by sodrooome** [comment]
request to review it @alvinisasi

File: app/Http/Controllers/OrderController.php [hunk 1]
```diff
@@ -13,15 +13,15 @@ public function create(){
...
-            'jenis_kertas' => 'required',
-            'jilid' => 'required',
-            'jasa_antar' => 'required',
-            'file' => 'required',
+            'rangkap' => 'required|integer|between:1,100',
+            'tinta' => 'required|max:50',
+            'jenis_kertas' => 'required|max:50',
+            'jilid' => 'required|max:100',
+            'jasa_antar' => 'required|boolean',
+            'file' => 'required|boolean|file|mimes:ppt,pptx,doc,docx,pdf,xls,xlsx|max:204800',
```
Comment (alvinisasi) [inline app/Http/Controllers/OrderController.php]: Thank you for this addition. I forget to add this requirement for document uploader.

File: app/Http/Controllers/OrderController.php [hunk 2]
```diff
@@ -13,15 +13,15 @@ public function create(){
...
             'takenTime' => 'required',
-            'notes' => 'required',
-            'agreement' => 'required'
+            'notes' => 'required|min:3|max:1000',
+            'agreement' => 'accepted'
```
Comment (alvinisasi) [inline app/Http/Controllers/OrderController.php:24]: What that's mean 'accepted'?

**Review by alvinisasi — APPROVED** [review]

File: app/Http/Controllers/OrderController.php (hunk 2 above)
Comment (hellodit) [reply app/Http/Controllers/OrderController.php:24]: Hii @alvinisasi according to laravel documentation accepted mean the field under validation must be yes, on, 1, or true.  You can read by your self [laravel doc](https://laravel.com/docs/7.x/validation#rule-accepted)

**Pull request merged**